pkg/pluginmanager/manager_test.go

```diff
@@ -268,7 +268,7 @@ func Test_InstallPlugin_InstalledPlugins_Central_Repo(t *testing.T) {
 	// Try installing the feature plugin which is targeted for k8s but requesting the TMC target
 	err = InstallStandalonePlugin("feature", "v0.2.0", configtypes.TargetTMC)
 	assertions.NotNil(err)
-	assertions.Contains(err.Error(), "unable to find plugin 'feature' for target 'mission-control'")
+	assertions.Contains(err.Error(), "unable to find plugin 'feature' with version 'v0.2.0' for target 'mission-control'")
```

Comment:
nit, if possible, in this file only two versions are referred to, "v1.6.0" and "v0.2.0", please convert them to constants and replace in all places.